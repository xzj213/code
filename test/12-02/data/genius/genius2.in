30
43 42
0.569 0.347 0.044 0.134 0.504 0.426 0.813 0.428 0.321 0.316 0.377 0.317 0.011 0.221 0.298 0.321 0.448 0.975 0.807 0.457 0.077 0.408 0.230 0.280 0.663 0.967 0.253 0.608 0.841 0.857 0.141 0.943 0.988 0.423 0.795 0.606 0.988 0.839 0.972 0.544 0.242 0.606 0.129 
0 1
1 2
2 3
2 4
0 5
0 6
2 7
1 8
8 9
0 10
0 11
6 12
10 13
1 14
6 15
10 16
2 17
2 18
15 19
17 20
10 21
5 22
10 23
3 24
13 25
22 26
18 27
26 28
8 29
22 30
26 31
18 32
7 33
22 34
6 35
16 36
12 37
4 38
26 39
12 40
40 41
36 42
49 48
0.293 0.962 0.547 0.310 0.489 0.787 0.259 0.431 0.695 0.911 0.242 0.449 0.370 0.853 0.227 0.764 0.551 0.603 0.630 0.754 0.759 0.310 0.779 0.137 0.899 0.518 0.438 0.819 0.867 0.792 0.430 0.460 0.910 0.023 0.939 0.208 0.925 0.946 0.317 0.034 0.012 0.164 0.496 0.393 0.726 0.322 0.187 0.769 0.308 
0 1
0 2
1 3
3 4
0 5
5 6
1 7
5 8
1 9
4 10
2 11
9 12
9 13
4 14
4 15
12 16
4 17
9 18
16 19
2 20
9 21
11 22
9 23
10 24
2 25
18 26
3 27
21 28
21 29
15 30
26 31
17 32
19 33
11 34
34 35
8 36
29 37
22 38
23 39
31 40
39 41
22 42
35 43
18 44
39 45
34 46
8 47
40 48
28 27
0.225 0.379 0.639 0.743 0.655 0.568 0.337 0.073 0.382 0.623 0.607 0.757 0.281 0.392 0.382 0.821 0.334 0.849 0.952 0.468 0.413 0.195 0.581 0.112 0.701 0.399 0.290 0.134 
0 1
1 2
0 3
1 4
2 5
0 6
3 7
2 8
5 9
8 10
6 11
0 12
11 13
5 14
14 15
13 16
15 17
16 18
17 19
5 20
17 21
20 22
21 23
10 24
10 25
1 26
0 27
31 30
0.426 0.614 0.454 0.273 0.416 0.980 0.180 0.010 0.090 0.746 0.807 0.384 0.799 0.389 0.469 0.755 0.451 0.553 0.964 0.988 0.087 0.481 0.010 0.424 0.740 0.861 0.950 0.394 0.199 0.273 0.514 
0 1
1 2
1 3
3 4
3 5
4 6
2 7
0 8
5 9
9 10
4 11
1 12
8 13
1 14
14 15
3 16
1 17
7 18
13 19
12 20
20 21
21 22
1 23
19 24
8 25
3 26
14 27
23 28
5 29
6 30
48 47
0.565 0.096 0.789 0.256 0.111 0.046 0.236 0.811 0.627 0.172 0.452 0.382 0.256 0.144 0.629 0.222 0.785 0.670 0.243 0.870 0.267 0.416 0.263 0.988 0.549 0.031 0.066 0.015 0.005 0.027 0.496 0.659 0.182 0.643 0.251 0.579 0.331 0.963 1.000 0.187 0.800 0.400 0.172 0.533 0.913 0.179 0.409 0.141 
0 1
0 2
1 3
3 4
4 5
0 6
5 7
3 8
8 9
2 10
2 11
4 12
0 13
10 14
14 15
13 16
11 17
7 18
9 19
13 20
2 21
1 22
5 23
22 24
9 25
14 26
15 27
22 28
0 29
0 30
30 31
0 32
20 33
33 34
8 35
1 36
6 37
16 38
29 39
37 40
39 41
31 42
37 43
11 44
4 45
6 46
3 47
33 32
0.720 0.930 0.456 0.334 0.950 0.275 0.581 0.021 0.751 0.382 0.147 0.488 0.119 0.647 0.595 0.156 0.969 0.826 0.000 0.879 0.858 0.842 0.007 0.289 0.847 0.823 0.643 0.768 0.031 0.315 0.202 0.758 0.617 
0 1
0 2
0 3
3 4
0 5
4 6
2 7
5 8
3 9
5 10
6 11
9 12
10 13
5 14
4 15
6 16
14 17
6 18
12 19
19 20
0 21
11 22
21 23
12 24
4 25
6 26
1 27
16 28
13 29
7 30
24 31
17 32
32 31
0.753 0.548 0.993 0.055 0.384 0.735 0.616 0.738 0.493 0.471 0.409 0.765 0.101 0.608 0.200 0.901 0.564 0.245 0.445 0.135 0.935 0.263 0.260 0.259 0.244 0.719 0.293 0.440 0.782 0.177 0.999 0.952 
0 1
0 2
1 3
0 4
2 5
3 6
0 7
1 8
0 9
7 10
5 11
10 12
0 13
11 14
9 15
4 16
15 17
8 18
10 19
15 20
9 21
8 22
4 23
23 24
10 25
25 26
15 27
25 28
16 29
19 30
28 31
24 23
0.118 0.695 0.164 0.812 0.583 0.596 0.174 0.588 0.502 0.142 0.140 0.576 0.100 0.107 0.259 0.984 0.706 0.174 0.516 0.135 0.327 0.778 0.566 0.673 
0 1
0 2
0 3
2 4
1 5
3 6
0 7
5 8
3 9
5 10
4 11
0 12
3 13
12 14
0 15
7 16
9 17
3 18
0 19
5 20
18 21
16 22
3 23
43 42
0.837 0.061 0.803 0.016 0.586 0.500 0.571 0.934 0.337 0.407 0.826 0.921 0.395 0.773 0.368 0.635 0.793 0.944 0.446 0.958 0.573 0.188 0.490 0.463 0.705 0.003 0.602 0.012 0.958 0.921 0.541 0.668 0.331 0.322 0.174 0.011 0.804 0.275 0.570 0.715 0.725 0.899 0.575 
0 1
0 2
0 3
0 4
1 5
4 6
6 7
2 8
6 9
0 10
8 11
10 12
10 13
7 14
4 15
4 16
13 17
3 18
16 19
3 20
9 21
15 22
22 23
5 24
17 25
19 26
19 27
2 28
18 29
24 30
25 31
25 32
24 33
5 34
12 35
31 36
34 37
35 38
27 39
33 40
6 41
1 42
44 43
0.732 0.811 0.902 0.163 0.133 0.207 0.274 0.394 0.355 0.708 0.364 0.363 0.179 0.814 0.081 0.634 0.950 0.088 0.621 0.608 0.754 0.004 0.846 0.197 0.929 0.400 0.474 0.637 0.043 0.878 0.045 0.777 0.213 0.703 0.319 0.418 0.347 0.008 0.153 0.963 0.599 0.243 0.252 0.646 
0 1
1 2
1 3
2 4
4 5
5 6
4 7
5 8
8 9
4 10
2 11
10 12
3 13
6 14
8 15
1 16
5 17
6 18
10 19
15 20
9 21
14 22
10 23
0 24
23 25
4 26
13 27
24 28
25 29
13 30
23 31
29 32
0 33
10 34
25 35
24 36
31 37
23 38
24 39
4 40
5 41
38 42
13 43
25 24
0.392 0.866 0.520 0.717 0.292 0.028 0.590 0.628 0.985 0.262 0.869 0.580 0.749 0.118 0.459 0.899 0.608 0.693 0.214 0.709 0.788 0.074 0.589 0.230 0.709 
0 1
0 2
2 3
2 4
3 5
2 6
1 7
0 8
0 9
2 10
7 11
2 12
5 13
7 14
0 15
9 16
4 17
15 18
12 19
2 20
9 21
10 22
0 23
5 24
32 31
0.621 0.983 0.637 0.104 0.350 0.900 0.993 0.995 0.840 0.018 0.135 0.634 0.885 0.216 0.243 0.676 0.503 0.861 0.763 0.051 0.232 0.933 0.706 0.288 0.762 0.550 0.278 0.360 0.357 0.655 0.430 0.911 
0 1
0 2
0 3
1 4
3 5
4 6
4 7
5 8
4 9
5 10
1 11
8 12
4 13
9 14
11 15
2 16
10 17
15 18
4 19
9 20
14 21
1 22
6 23
12 24
17 25
16 26
20 27
24 28
11 29
11 30
30 31
47 46
0.766 0.708 0.705 0.793 0.548 0.022 0.294 0.499 0.317 0.340 0.778 0.197 0.938 0.444 0.702 0.463 0.404 0.317 0.629 0.434 0.856 0.640 0.145 0.123 0.051 0.008 0.596 0.250 0.426 0.452 0.448 0.107 0.834 0.011 0.378 0.531 0.584 0.754 0.644 0.018 0.087 0.552 0.903 0.736 0.697 0.605 0.528 
0 1
0 2
2 3
1 4
1 5
5 6
0 7
3 8
3 9
4 10
0 11
4 12
4 13
11 14
10 15
2 16
13 17
12 18
17 19
19 20
16 21
2 22
14 23
7 24
11 25
23 26
23 27
25 28
0 29
11 30
29 31
2 32
21 33
23 34
33 35
24 36
25 37
10 38
6 39
30 40
14 41
17 42
33 43
15 44
23 45
20 46
37 36
0.588 0.228 0.405 0.000 0.852 0.593 0.466 0.062 0.446 0.246 0.428 0.594 0.969 0.687 0.572 0.500 0.620 0.461 0.973 0.186 0.569 0.003 0.172 0.648 0.983 0.425 0.574 0.161 0.168 0.578 0.535 0.052 0.867 0.271 0.591 0.735 0.140 
0 1
0 2
2 3
0 4
2 5
0 6
2 7
3 8
5 9
5 10
10 11
0 12
10 13
7 14
7 15
4 16
5 17
13 18
13 19
1 20
14 21
13 22
21 23
16 24
5 25
12 26
7 27
4 28
24 29
7 30
28 31
3 32
4 33
10 34
32 35
0 36
48 47
0.987 0.303 0.770 0.567 0.786 0.131 0.870 0.643 0.777 0.446 0.631 0.596 0.659 0.599 0.748 0.609 0.847 0.195 0.666 0.717 0.779 0.542 0.690 0.411 0.322 0.935 0.457 0.612 0.717 0.548 0.089 0.609 0.333 0.048 0.853 0.612 0.272 0.175 0.392 0.327 0.391 0.493 0.243 0.351 0.951 0.146 0.349 0.575 
0 1
0 2
1 3
1 4
3 5
3 6
4 7
4 8
7 9
9 10
2 11
7 12
3 13
8 14
12 15
1 16
2 17
10 18
2 19
5 20
8 21
16 22
5 23
22 24
1 25
18 26
21 27
18 28
22 29
18 30
13 31
9 32
19 33
21 34
1 35
24 36
3 37
10 38
2 39
1 40
23 41
41 42
32 43
11 44
36 45
22 46
44 47
48 47
0.985 0.103 0.839 0.170 0.234 0.566 0.175 0.312 0.393 0.155 0.754 0.898 0.066 0.114 0.052 0.182 0.365 0.803 0.528 0.000 0.483 0.223 0.725 0.270 0.049 0.720 0.946 0.495 0.260 0.792 0.529 0.201 0.712 0.828 0.330 0.927 0.615 0.930 0.023 0.182 0.123 0.337 0.210 0.168 0.752 0.776 0.710 0.136 
0 1
0 2
0 3
2 4
1 5
1 6
2 7
6 8
1 9
2 10
5 11
4 12
9 13
2 14
6 15
3 16
0 17
13 18
16 19
4 20
9 21
11 22
11 23
19 24
5 25
13 26
19 27
17 28
4 29
19 30
22 31
13 32
23 33
14 34
33 35
12 36
34 37
11 38
32 39
32 40
6 41
4 42
15 43
11 44
38 45
33 46
10 47
26 25
0.819 0.206 0.024 0.953 0.626 0.061 0.470 0.532 0.337 0.367 0.527 0.078 0.807 0.444 0.900 0.774 0.292 0.430 0.829 0.906 0.253 0.604 0.247 0.829 0.366 0.816 
0 1
1 2
1 3
0 4
0 5
2 6
1 7
4 8
5 9
4 10
10 11
2 12
10 13
11 14
4 15
14 16
13 17
3 18
4 19
3 20
13 21
19 22
21 23
0 24
5 25
36 35
0.124 0.605 0.105 0.572 0.386 0.916 0.533 0.154 0.959 0.021 0.653 0.427 0.698 0.982 0.973 0.194 0.063 0.452 0.101 0.480 0.945 0.570 0.872 0.821 0.241 0.976 0.783 0.160 0.820 0.576 0.725 0.897 0.608 0.300 0.209 0.880 
0 1
1 2
1 3
1 4
4 5
5 6
0 7
4 8
5 9
4 10
10 11
7 12
7 13
1 14
12 15
9 16
15 17
12 18
10 19
13 20
1 21
3 22
12 23
20 24
15 25
23 26
20 27
22 28
11 29
25 30
11 31
12 32
4 33
25 34
25 35
43 42
0.682 0.353 0.870 0.627 0.431 0.909 0.497 0.149 0.402 0.986 0.491 0.183 0.662 0.843 0.347 0.029 0.457 0.316 0.874 0.451 0.653 0.831 0.936 0.532 0.559 0.467 0.706 0.277 0.214 0.031 0.617 0.080 0.926 0.129 0.353 0.670 0.651 0.461 0.731 0.274 0.776 0.301 0.425 
0 1
0 2
1 3
3 4
1 5
5 6
5 7
4 8
2 9
2 10
4 11
8 12
8 13
4 14
8 15
11 16
5 17
14 18
12 19
9 20
12 21
3 22
7 23
19 24
1 25
2 26
10 27
9 28
26 29
2 30
17 31
19 32
22 33
21 34
14 35
27 36
27 37
8 38
8 39
27 40
17 41
31 42
42 41
0.310 0.527 0.436 0.113 0.974 0.205 0.751 0.629 0.707 0.846 0.142 0.075 0.160 0.139 0.527 0.956 0.653 0.450 0.466 0.516 0.124 0.235 0.449 0.876 0.850 0.993 0.441 0.106 0.210 0.339 0.017 0.854 0.727 0.908 0.446 0.270 0.912 0.401 0.625 0.876 0.538 0.727 
0 1
0 2
1 3
0 4
0 5
3 6
1 7
4 8
1 9
4 10
2 11
3 12
7 13
13 14
9 15
0 16
13 17
2 18
6 19
16 20
13 21
18 22
5 23
13 24
10 25
8 26
25 27
13 28
14 29
0 30
21 31
23 32
6 33
21 34
2 35
31 36
7 37
26 38
15 39
18 40
35 41
22 21
0.706 0.263 0.476 0.325 0.801 0.541 0.603 0.435 0.013 0.351 0.637 0.798 0.323 0.461 0.622 0.480 0.633 0.311 0.384 0.300 0.963 0.919 
0 1
1 2
2 3
3 4
3 5
0 6
5 7
5 8
5 9
0 10
5 11
10 12
12 13
12 14
12 15
1 16
0 17
16 18
7 19
10 20
4 21
24 23
0.876 0.482 0.770 0.589 0.879 0.129 0.244 0.868 0.696 0.465 0.383 0.286 0.161 0.391 0.975 0.272 0.260 0.280 0.700 0.482 0.033 0.954 0.369 0.353 
0 1
1 2
2 3
3 4
4 5
1 6
4 7
6 8
8 9
7 10
10 11
4 12
8 13
2 14
10 15
7 16
10 17
8 18
6 19
16 20
3 21
16 22
12 23
20 19
0.109 0.158 0.101 0.421 0.229 0.707 0.291 0.811 0.115 0.812 0.887 0.038 0.214 0.534 0.465 0.947 0.331 0.179 0.045 0.945 
0 1
1 2
1 3
2 4
1 5
2 6
6 7
3 8
4 9
7 10
6 11
2 12
5 13
3 14
3 15
7 16
0 17
13 18
11 19
24 23
0.516 0.778 0.630 0.146 0.518 0.842 0.213 0.873 0.883 0.012 0.327 0.238 0.366 0.897 0.732 0.803 0.500 0.092 0.479 0.955 0.548 0.800 0.592 0.558 
0 1
0 2
2 3
3 4
4 5
0 6
3 7
2 8
5 9
9 10
6 11
1 12
10 13
2 14
8 15
6 16
10 17
17 18
0 19
2 20
0 21
13 22
10 23
35 34
0.148 0.811 0.487 0.865 0.142 0.980 0.206 0.123 0.247 0.413 0.115 0.739 0.878 0.389 0.517 0.007 0.616 0.692 0.627 0.929 0.419 0.574 0.714 0.788 0.813 0.073 0.148 0.369 0.821 0.356 0.020 0.576 0.910 0.072 0.670 
0 1
0 2
2 3
1 4
3 5
1 6
0 7
6 8
7 9
1 10
7 11
10 12
9 13
0 14
12 15
14 16
14 17
7 18
3 19
19 20
17 21
21 22
11 23
22 24
22 25
4 26
25 27
18 28
27 29
14 30
7 31
31 32
16 33
16 34
34 33
0.234 0.323 0.652 0.945 0.862 0.208 0.181 0.057 0.159 0.016 0.891 0.347 0.164 0.997 0.083 0.406 0.871 0.071 0.465 0.982 0.779 0.478 0.725 0.849 0.493 0.743 0.312 0.139 0.598 0.250 0.808 0.738 0.283 0.929 
0 1
0 2
1 3
3 4
1 5
4 6
2 7
0 8
0 9
5 10
3 11
2 12
0 13
4 14
12 15
14 16
12 17
14 18
15 19
19 20
5 21
11 22
3 23
10 24
9 25
22 26
15 27
27 28
19 29
20 30
29 31
30 32
24 33
26 25
0.369 0.121 0.676 0.301 0.410 0.428 0.464 0.653 0.731 0.202 0.294 0.453 0.078 0.966 0.501 0.651 0.356 0.731 0.804 0.881 0.512 0.036 0.047 0.548 0.705 0.961 
0 1
0 2
1 3
3 4
3 5
5 6
4 7
2 8
4 9
2 10
6 11
3 12
11 13
1 14
9 15
6 16
0 17
5 18
0 19
4 20
20 21
16 22
19 23
3 24
18 25
22 21
0.389 0.574 0.920 0.397 0.651 0.538 0.812 0.734 0.700 0.426 0.330 0.464 0.267 0.738 0.997 0.276 0.255 0.654 0.871 0.229 0.805 0.687 
0 1
0 2
0 3
2 4
3 5
3 6
0 7
1 8
6 9
0 10
1 11
11 12
7 13
6 14
2 15
2 16
9 17
14 18
10 19
4 20
16 21
23 22
0.175 0.711 0.040 0.793 0.398 0.182 0.088 0.425 0.585 0.839 0.940 0.254 0.358 0.869 0.875 0.681 0.448 0.085 0.316 0.965 0.363 0.370 0.972 
0 1
1 2
1 3
1 4
4 5
1 6
1 7
6 8
1 9
5 10
6 11
1 12
4 13
11 14
14 15
9 16
10 17
10 18
1 19
17 20
7 21
6 22
22 21
0.533 0.121 0.547 0.665 0.312 0.575 0.018 0.142 0.659 0.109 0.594 0.694 0.140 0.048 0.716 0.033 0.921 0.513 0.461 0.370 0.775 0.209 
0 1
1 2
0 3
2 4
3 5
3 6
1 7
2 8
5 9
9 10
7 11
0 12
3 13
11 14
8 15
14 16
8 17
3 18
13 19
5 20
17 21
