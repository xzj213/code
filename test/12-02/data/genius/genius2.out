2 0 6
2 0 1
2 1 2
2 0 1
2 0 2
2 0 1
2 0 1
2 0 15
2 0 2
2 0 1
2 0 1
2 0 1
2 0 1
2 0 4
2 0 1
2 0 1
2 0 1
2 1 3
2 0 1
2 0 4
2 0 6
2 0 1
2 3 15
2 0 1
2 1 6
2 0 13
2 0 2
2 0 2
2 1 3
2 0 3
